<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_device_tree">
     <attribute name="title">
      <string>current devices</string>
     </attribute>
     <widget class="QTreeWidget" name="device_tree">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>481</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>200</number>
      </attribute>
      <column>
       <property name="text">
        <string>device</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>vendor</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_rule_editor">
     <attribute name="title">
      <string>rule editor</string>
     </attribute>
     <widget class="QTableWidget" name="rule_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>511</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>30</width>
        <height>183</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="rule_operation_layout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="rule_existence_layout">
         <item>
          <widget class="QToolButton" name="rule_add_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="rule_remove_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="rule_order_layout">
         <item>
          <widget class="QToolButton" name="rule_up_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="rule_down_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="server_upload_button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Gemini active :</string>
    </property>
   </widget>
   <widget class="QLabel" name="server_icon_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons/icons.qrc">:/icons/enabled</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
